src/metrics/filters/tags_filter_test.go

```diff
@@ -88,6 +88,26 @@ func TestEmptyTagsFilterMatches(t *testing.T) {
 	require.True(t, matches)
 }
 
+func TestNonexistTagsFilterMatches(t *testing.T) {
+	filters := map[string]FilterValue{
+		"tagName1": FilterValue{Pattern: "tagValue1"},
```

Comment:
Can we also add another test case for duplicate tag values, since we are going to append tags to the original filter string?
E.g. 
```
"filter": "__name__:logdaemon_aggregatedRateLimitExceeded_count project:* proxy:kinesis2prom project:*"
```

Reply:
No, i won't append if `project` already exists